src/aten_sink.erl

```diff
@@ -78,6 +86,15 @@ code_change(_OldVsn, State, _Extra) ->
 %%% Internal functions
 %%%===================================================================
 
+maybe_monitor_node(Node, Monitors0) ->
+    IsMonitored = maps:get(Node, Monitors0, false),
+    case IsMonitored of
+        true -> Monitors0;
+        false ->
+            ct:pal("aten_sink node ~p monitoring node: ~p~n", [node(), Node]),
```

Comment:
This is weird @kjnilsson - I can see in the test output that the slave node is only monitored once, yet you can see in the output that the process receives hundreds of `nodedown` messages. I must be missing something. Also, two tests fail :-/